scriptworker/test/test_production.py

```diff
@@ -0,0 +1,187 @@
+}, {
+    "name": "reference-browser nightly",
+    "index": "project.mobile.reference-browser.v2.nightly.latest",
+    "task_type": "build",
+    "cot_product": "mobile",
+}, {
+    "name": "reference-browser master raptor aarch64",
+    "index": "project.mobile.reference-browser.v2.branch.master.latest.raptor.aarch64",
+    "task_type": "build",
+    "cot_product": "mobile",
+}, {
+    "name": "focus nightly",
+    "index": "project.mobile.focus.nightly.latest",
+    "task_type": "build",
+    "cot_product": "mobile",
+}))
+@pytest.mark.asyncio
+async def test_verify_production_cot(branch_context):
+    index = Index(options={'rootUrl': DEFAULT_CONFIG['taskcluster_root_url']})
+    queue = Queue(options={'rootUrl': DEFAULT_CONFIG['taskcluster_root_url']})
+
+    async def get_task_id_from_index(index_path):
+        res = await index.findTask(index_path)
+        return res['taskId']
+
+    async def get_completed_task_info_from_labels(decision_task_id, label_to_task_type):
+        label_to_taskid = await queue.getLatestArtifact(
+            decision_task_id, "public/label-to-taskid.json"
+        )
+        task_info = {}
+        for re_label, task_type in label_to_task_type.items():
+            r = re.compile(re_label)
+            for label, task_id in label_to_taskid.items():
+                if r.match(label):
+                    status = await queue.status(task_id)
+                    # only run verify_cot against tasks with completed deps.
+                    if status['status']['state'] in ('completed', 'running', 'pending', 'failed'):
+                        task_info[task_id] = task_type
+                        break
+            else:
+                log.warning(
+                    "Not running verify_cot against {} {} because there are no elegible completed tasks".format(
+                        decision_task_id, task_type
+                    )
+                )
+        return task_info
+
+    async def verify_cot(name, task_id, task_type, cot_product):
+        log.info("Verifying {} {} {}...".format(name, task_id, task_type))
+        context.task = await queue.task(task_id)
+        cot = ChainOfTrust(context, task_type, task_id=task_id)
+        await verify_chain_of_trust(cot)
+
+    async with get_context({'cot_product': branch_context['cot_product']}) as context:
+        context.queue = queue
+        task_id = await get_task_id_from_index(branch_context['index'])
+        assert task_id, "{}: Can't get task_id from index {}!".format(
+            branch_context['name'], branch_context['index']
+        )
+        if branch_context.get('task_label_to_task_type'):
+            task_info = await get_completed_task_info_from_labels(
+                task_id, branch_context['task_label_to_task_type']
+            )
+            for task_id, task_type in task_info.items():
+                name = "{} {}".format(branch_context['name'], task_type)
+                await verify_cot(name, task_id, task_type)
```

Comment:
Do these work for you? I get

`E               TypeError: verify_cot() missing 1 required positional argument: 'cot_product'`

Seems like a simple fix.